pkg/imgpkg/cmd/copy.go

```diff
@@ -77,9 +75,12 @@ func (o *CopyOptions) Run() error {
 	logger := ctlimg.NewLogger(os.Stderr)
 	prefixedLogger := logger.NewPrefixedWriter("copy | ")
 
-	registry, err := ctlimg.NewRegistry(o.RegistryFlags.AsRegistryOpts(), imagelayers.NewImageLayerWriterCheck(o.IncludeNonDistributableFlag.IncludeNonDistributable))
+	opts := o.RegistryFlags.AsRegistryOpts()
```

Comment:
opts -> registryOpts to make it more obvious what this is in context of copy cmd.

Reply:
ok done.